gbbs/benchmarks/BatchDynamicConnectivity/EulerTourTree/ETTree.h

```diff
@@ -593,33 +593,18 @@ struct ETTree {
     }
     
     template <class KY, class VL, class HH>
-    sequence<sequence<sequence<std::pair<uintE, uintE>>>> batch_sub_tree_sum(sequence<std::pair<uintE,uintE>> v_p,
-            pbbslib::sparse_table<KY, VL, HH> edge_index_table)
+    sequence<sequence<sequence<std::pair<uintE, uintE>>>> batch_sub_tree_sum(const sequence<std::pair<uintE,uintE>>& v_p,
+            const pbbslib::sparse_table<KY, VL, HH>& edge_index_table)
     {
 
-
         size_t num_queries = v_p.size();
```

Comment:
```suggestion
        std::size_t const num_queries = v_p.size();
```
This value shouldn't change in this function, correct?